swarm/network/stream/testing/testing.go

```diff
@@ -162,8 +164,11 @@ func NewSimulation(conf *RunConfig) (*Simulation, func(), error) {
 	// set nodes number of Stores available
 	stores, storeTeardown, err := SetStores(addrs...)
 	teardown = func() {
+		log.Trace("simulation: teardown net")
 		net.Shutdown()
+		log.Trace("simulation: teardown adapter")
 		adapterTeardown()
+		log.Trace("simulation: teardown store")
```

Comment:
@janos Too much logging, I guess these were just meant for debugging.

Reply:
👍